.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,17 @@
+<!-- Put a description of what this PR is for here, along with any references to issues that this resolves or contributes to -->
+
+
+
+
+<!-- Replace each [ ] with [X] to check it. Switch to the preview view to make it easier to click on links -->
+If this PR is for Terraform, I acknowledge that I have:
```

Comment:
Is it worth mentioning that completing these steps will speed up the review process? Or that a reviewer won't consider reviewing until the steps are completed/acknowledged in some way?

Reply:
Done.